src/lib/list/_list-theme.scss

```diff
@@ -46,7 +46,8 @@
     }
 
     .mat-subheader {
-      @include mat-typography-level-to-styles($config, body-2);
+      font: mat-font-weight($config, body-2) mat-font-size($config, body-2)
```

Comment:
Let's do that. Also the `font-family` could be set once on the `.mat-list` and the subheader can inherit from there instead.